comparisonModelsExperiment/script/measures.py

```diff
@@ -148,34 +182,107 @@ def evaluate_metrics(confusion_matrix_resolver, df, baseline_name):
     print("MCC = ", mcc)
     print("BA = ", ba)
 
+    return mcc, ba
+
+
+def generate_charts(classifiers, metric, values):
+    fig, ax = plt.subplots()
+    index = range(len(classifiers))
+
+    ax.bar(index, values)
+    ax.set_xlabel('Classifiers')
+    ax.set_ylabel(metric)
+    ax.set_title(f'Comparison of {metric} between Classifiers')
+    ax.set_xticks(index)
+    ax.set_xticklabels(classifiers, rotation=45)
+
+    # Find the classifier with the highest MCC
+    best_classifier = classifiers[values.index(max(values))]
+    ax.axhline(max(values), color='r', linestyle='--',
+               label=f'Best Classifier: {best_classifier}')
+    ax.legend()
+
+    plt.tight_layout()
+    plt.show()
```

Comment:
We also need to save this to file, because it cannot be opened on google colab (reproduction for others).